### PR TITLE
CLI automate usergroup subcommand

### PR DESCRIPTION
**Pull request by lpramuk**
CLI automate usergroup subcommand

+ class representing own hammer user-group sub-command
+ factory methods make_usergroup(), make_usergroup_external()

(Part of #3072)

Test class is still to be implemented....but that's another PR

### CODE REVIEW

File: robottelo/cli/usergroup.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    delete                        Delete a user group
+    external                      View and manage external user groups
+    info                          Show a user group
+    list                          List all user groups
+    remove-role                   Remove a user role
+    remove-user                   Disassociate an user
+    remove-user-group             Disassociate an user group
+    update                        Update a user group
+"""
+
```
Comment (elyezer) [inline robottelo/cli/usergroup.py]: This blank line can be droped.

File: robottelo/cli/usergroup.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+from robottelo.cli.base import Base
+
+
+class UserGroup(Base):
+    """
```
Comment (elyezer) [inline robottelo/cli/usergroup.py]: One line docstrings should follow `"""One line."""`

**Comment by elyezer** [comment]
ACK, pending nitpicks for the CLI helper classes.

**Comment by lpramuk** [comment]
Nitpicks resolved.
Added factory methods.

**Comment by oshtaier** [comment]
Looks good
ACK

**Comment by oshtaier** [comment]
There are no tests proving that methods will work properly, but I think that fact will be verified really soon.

**Comment by elyezer** [comment]
:+1:

**Pull request merged**